src/views/object/ObjectDetails.vue

```diff
@@ -33,13 +33,39 @@ import { objectStore, navigationStore } from '../../store/store.js'
 				<p>
 					{{ JSON.stringify(objectStore.objectItem.object, null, 2) }}
 				</p>
+
+				<div class="tabContainer">
+					<BTabs content-class="mt-3" justified>
+						<BTab title="Logs">
+							<div v-if="false && logs.length > 0">
+								<NcListItem v-for="(log, key) in logs"
+									:key="key"
+									:name="log.title"
+									:bold="false"
+									:force-display-actions="true">
+									<template #icon>
+										<PostOutline disable-menu
+											:size="44" />
+									</template>
+									<template #subname>
+										{{ log.description }}
+									</template>
+								</NcListItem>
+							</div>
+							<div v-if="true || logs.length === 0">
```

Comment:
If you set `.length === 0` to `!.length` in all files and add `class="tabPanel"`  in all files it all wil look a lot better